web/app/firebase/config.tsx

```diff
@@ -24,27 +24,27 @@ const isLocalhost = () => {
   return process && process.env.NODE_ENV === "development";
 };
 
-if (isLocalhost()) {
-  database = getDatabase();
-  storage = getStorage();
-  connectDatabaseEmulator(database, "localhost", 9000);
-  connectStorageEmulator(storage, "localhost", 9199);
-  firebase.functions().useEmulator("localhost", 5001);
-  firebase.auth().useEmulator("http://localhost:9099");
-  functions = firebase.functions();
-  if (
-    typeof sessionStorage != "undefined" &&
-    !sessionStorage.getItem("givenWarning")
-  ) {
-    alert(
-      "Initializing in emulator mode. If you aren't a developer, contact support@nuft_getdomain.com immediately."
-    );
-    sessionStorage.setItem("givenWarning", "true");
-  }
-} else {
```

Comment:
you should be using a js runtime to host server (npm run dev or bun run dev) plus firebase emulators, do you have a different strategy

Reply:
the emulator wasn't working for me so I just commented them out and I was iterating with the ACM ICAIF production database for speed. There's no real user data in that db right now

Reply:
<img width="787" alt="Screenshot 2023-10-24 at 1 47 52 PM" src="https://github.com/FinRL-Contest/ACM_ICAIF_2023/assets/79772347/81629f19-d48d-45bf-84a1-14cd39e94874">